README.md

```diff
@@ -0,0 +1,198 @@
+--------------------------------------------------
+
+Sometimes it can be useful to first run some tests on the client, then after those are done, run some tests on the server to check whether the client calls correctly affected the backend storage. This can be done by interleaving client-side sub-tests with server-side sub-tests. We take the previous async test example and add a server-side sub-test between the existing two:
+
+```coffeescript
+  testClientFoo:
+    [
+      ->
+        # Call the first method.
+        Meteor.call 'first', 'argument', @expect (error, result) =>
+          @assertIsUndefined error, "Error while calling first: #{ error }"
+    ,
+      @runOnServer ->
+        # Check if the first method really cleared everything in Foo collection.
+        @assertEqual Foo.documents.find().count(), 0
+    ,
+      ->
+        # Call the second method.
+        Meteor.call 'second', 'argument', @expect (error, result) =>
+          @assertIsUndefined error, "Error while calling second: #{ error }"
+    ]
+```
+
+After the `first` method call completes, the second sub-test will be executed on the server and all assertions will be propagated back to the client.
+
+Passing variables from server-side tests to client-side tests
+-------------------------------------------------------------
+
+Sometimes there is the need of passing variables from server-side tests for use in client-side tests, usually when defining fixtures in `setUp` methods. Consider this example:
+
+```coffeescript
+  setUpServer: =>
+    # Initialize the database.
+    Foo.documents.remove {}
+
+    # Create a test document.
+    @testDocumentId = Foo.documents.insert
+      bar: true
+
+  testClientRemoval: =>
+    Meteor.call 'remove', @testDocumentId, @expect (error, result) =>
+      @assertIsUndefined error, "Error while remove: #{ error }"
+```
+
+So before the test starts we create a test fixture on the server and would then like to reference its `_id` on the client. The problem is that this will not work as the test case instance on the server differs from the one on the client and `@testDocumentId` will not be available there. In order to address this, classy tests support passing specific variables from server-side tests to client-side tests using `@get` and `@set` methods. In order to fix the above example we can do:
+
+```coffeescript
+  setUpServer: =>
+    # Initialize the database.
+    Foo.documents.remove {}
+
+    # Create a test document.
+    testDocumentId = Foo.documents.insert
+      bar: true
+
+    # Pass variable to client-side tests.
+    @set 'testDocumentId', testDocumentId
+
+  testClientRemoval: =>
+    Meteor.call 'remove', @get('testDocumentId'), @expect (error, result) =>
+      @assertIsUndefined error, "Error while remove: #{ error }"
+```
+
+In the background, the test framework will seamlessly transfer the variables between the tests. Note that as variables are transferred via DDP, they must be EJSON serializable.
```

Comment:
This currently works only server->client. Should you mention this limitation? Or just lift it?